src/averageAges.js

```diff
@@ -20,6 +20,15 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const filteredMen = people.filter(
+    person => person.sex === 'm'
+    && (!century || century === Math.ceil(person.died / 100)));
```

Comment:
Пропоную записувати в такому форматі
```suggestion
  const filteredMen = people.filter(
     century ? firstCallback : secondCallback
  )